### PR TITLE
feat(rpc/v0.2): implement `starknet_getClassHashAt`

### LINKED ISSUE #596: `starknet_getClassHashAt`

### PR DESCRIPTION
**Pull request by Mirko-von-Leipzig**
Relevant specification [link](https://github.com/starkware-libs/starknet-specs/blob/v0.2.0/api/starknet_api_openrpc.json#L277-L313).

Closes #596.

### CODE REVIEW

File: crates/pathfinder/src/rpc/v02.rs [hunk 1]
```diff
@@ -151,6 +151,11 @@ where
 // Registers all methods for the v0.2 API
 pub fn register_all_methods(module: &mut jsonrpsee::RpcModule<RpcContext>) -> anyhow::Result<()> {
     register_method_with_no_input(module, "starknet_chainId", method::chain_id::chain_id)?;
+    register_method(
+        module,
+        "starknet_getClassHashAt",
+        method::get_class_hash_at,
+    );
```
Comment (sergey-melnychuk) [inline crates/pathfinder/src/rpc/v02.rs:158]: `?`? :)

File: crates/pathfinder/src/rpc/v02.rs [hunk 2]
```diff
@@ -151,6 +151,11 @@ where
 // Registers all methods for the v0.2 API
 pub fn register_all_methods(module: &mut jsonrpsee::RpcModule<RpcContext>) -> anyhow::Result<()> {
     register_method_with_no_input(module, "starknet_chainId", method::chain_id::chain_id)?;
+    register_method(
+        module,
+        "starknet_getClassHashAt",
+        method::get_class_hash_at,
```
Comment (sergey-melnychuk) [inline crates/pathfinder/src/rpc/v02.rs:157]: This is likely the reason why build fails: `method::get_class_hash_at::get_class_hash_at` should do.

File: crates/pathfinder/src/rpc/v02/method/get_class_hash_at.rs [hunk 3]
```diff
@@ -0,0 +1,272 @@
...
+
+crate::rpc::error::generate_rpc_error_subset!(GetClassHashAtError: BlockNotFound, ContractNotFound);
+
+#[derive(serde::Deserialize, Debug, PartialEq, Eq)]
+pub struct GetClassHashAtInput {
+    block_id: BlockId,
+    contract_address: ContractAddress,
+}
+
+#[allow(dead_code)]
```
Comment (sergey-melnychuk) [inline crates/pathfinder/src/rpc/v02/method/get_class_hash_at.rs:16]: This seems unnecessary, as it is not a dead code.

**Review by sergey-melnychuk — APPROVED** [review]
LGTM (Trivial changes necessary to fix the build).

**Comment by Mirko-von-Leipzig** [comment]
> LGTM (Trivial changes necessary to fix the build).

Thanks; that's what I get for pushing and then leaving for the day :D

**Review by kkovaacs — APPROVED** [review]
LGTM

**Pull request merged**